images/linux/ubuntu1604.json

```diff
@@ -186,7 +186,8 @@
                 "{{template_dir}}/scripts/installers/vcpkg.sh",
                 "{{template_dir}}/scripts/installers/zeit-now.sh",
                 "{{template_dir}}/scripts/installers/updatepath.sh",
-                "{{template_dir}}/scripts/installers/dpkg-config.sh"
+                "{{template_dir}}/scripts/installers/dpkg-config.sh",
+                "{{template_dir}}/scripts/installers/homebrew.sh"
```

Comment:
I think we should add homebrew installation at the begin of this list because potentially it can be used to install other software